catalog/imaging/imaging-actionprovider-chip/src/test/java/org/codice/alliance/imaging/chip/actionprovider/TestImageChipActionProvider.java

```diff
@@ -98,21 +105,21 @@ public void testGetActionsNullMetacard() {
 
     @Test
     public void testGetActionsCanNotHandleMetacard() {
-        metacard.setLocation(null);
-        assertThat(imagingChipActionProvider.getActions(metacard), hasSize(0));
+        imageMetacard.setLocation(null);
+        assertThat(imagingChipActionProvider.getActions(imageMetacard), hasSize(0));
     }
 
     @Test
     public void testGetActionsUnsupportedProtocolOnMetacard() {
         String protocol = SystemBaseUrl.getProtocol();
         System.setProperty("org.codice.ddf.system.protocol", "udp://");
-        assertThat(imagingChipActionProvider.getActions(metacard), hasSize(0));
+        assertThat(imagingChipActionProvider.getActions(imageMetacard), hasSize(0));
```

Comment:
I'm unclear why changing the protocol does this. Nothing that I could see in the implementation of the action provider should care about this and it's no javadocs indicating that the protocol has an effect.